pkg/ddc/jindocache/transform.go

```diff
@@ -816,6 +820,41 @@ func (e *JindoCacheEngine) transformFuse(runtime *datav1alpha1.JindoRuntime, val
 	value.Fuse.CriticalPod = common.CriticalFusePodEnabled()
 }
 
+func (e *JindoCacheEngine) transformFuseMetrics(runtime *datav1alpha1.JindoRuntime, value *Jindo) error {
+	var userDefinedPort int = -1
+	for _, arg := range value.Fuse.Args {
+		// user may explicitly set a metrics port in fuse args
+		if strings.HasPrefix(arg, "-ometrics_port=") {
+			if port, err := strconv.ParseInt(strings.TrimPrefix(arg, "-ometrics_port="), 10, 32); err != nil {
+				return errors.Wrap(err, "failed to parse port from %s transformFuseMetrics()")
```

Comment:
I think the error message formatting misses `arg`. It should like as below:

```golang
return errors.Wrap(err, fmt.Sprintf("failed to parse port from %s", arg))
```

Reply:
Done.